mailparser/mailparser.py

```diff
@@ -226,8 +226,12 @@ def message_id(self):
 
     @property
     def to_(self):
-        return decode_header_part(
-            self._message.get('to', self._message.get('delivered-to')))
+        to_ = self._message.get('to')        
+        if not to_:
+            return None
+        else:
```

Comment:
Your code was complete. The method *.get* return *None* if you can get *to*. In your *else* you will  alway have *self._message.get('to')*. Can you give me a gist (https://gist.github.com/) with an example of mail (that %1)?